package ch16.sec01.exam01;
public class ExceotionEx03 {
	public static void main(String[] args) {
		throw new RuntimeException();		//Exception을 발생시킨다.

	}

}
